cmd/aries-js-worker/README.md

```diff
@@ -27,16 +27,90 @@ Go to [`vue-framework-go`](./vue-aries-framework-go) for a sample agent built wi
 
 View [`index.html`](./index.html) for examples.
 
-**Important:** Make sure the web server adds the appropriate headers when serving the compressed `aries-js-worker.wasm.gz` file:
+### Important - Serving the Assets
+
+`aries-js-worker` loads some assets at runtime: the web assembly binary and a couple of JS scripts. These assets are
+located in the `dist/assets` directory (if you `npm install` it, you'll find them in
+`./node_modules/@hyperledger/aries-framework-go/dist/assets`).
+
+Things that need to work if you are to use `aries-js-worker` on the client side.
```

Comment:
How do you enable webpack dev server (and vue CLI)?
@llorllale @rolsonquadras

Reply:
checking

Reply:
Two issues here:

1. Serve compressed file with correct headers
2. Server request for `.wasm` with the `.wasm.gz` file

I don't see how to fix either of these yet. A potential solution is to inject some middleware (eg. [`before`](https://webpack.js.org/configuration/dev-server/#devserverbefore)) but that didn't seem to work when I fiddled around with it a few days ago.

I can work around this right now by having `npm run server` uncompress the wasm.

Reply:
So should we document the workaround for now?

Reply:
Sure

Reply:
done

Reply:
@llorllale Do you have a link to that script from these instructions? It's not quite obvious from the text that there are extra step required.

Reply:
From this README or the one in vue-aries-framework-go ?

Reply:
I had been thinking this README.

However, maybe it makes sense in the vue-aries-framework-go README and also a link from this README mentioning Vue CLI and Webpack dev server.

Reply:
That script is uncompressing the wasm and then serving it...not recommended for production. It's executed only for `npm run serve` which is supposed to be for development purposes.

Reply:
Right - it's still useful information for developers ... who will be confused why things just don't work :).

Reply:
(so maybe put the information in the other README and a small blurb for vue/webpack developers to know to look there).

Reply:
Added a couple of notes to both READMEs.